From a469083445485202219f9a43c9caaeca345b11cd Mon Sep 17 00:00:00 2001
From: Dan Kephart <dan.kephart@lairdtech.com>
Date: Wed, 30 Oct 2013 11:25:03 -0400
Subject: [PATCH] Fix for channels 157, 161, and 165 not working with DFS off
 and FCC

This coupled with firmware fix in 3.4.0.77 allows the firmware
to use 65 channels for all WMI commands that allow setting or getting
a specific channel set.
---
 drivers/net/wireless/ath/ath6kl/wmi.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath6kl/wmi.h b/drivers/net/wireless/ath/ath6kl/wmi.h
index 42d2751..64eafa1 100644
--- a/drivers/net/wireless/ath/ath6kl/wmi.h
+++ b/drivers/net/wireless/ath/ath6kl/wmi.h
@@ -1233,7 +1233,7 @@ enum wmi_dfs_mode {
 	DFS_ENABLED,
 };
 
-#define WMI_MAX_CHANNELS        32
+#define WMI_MAX_CHANNELS        64
 
 struct wmi_channel_params_cmd {
     u8     reserved1;
-- 
1.8.3.msysgit.0

